assets/js/modules/pagespeed-insights/components/common/PageSpeedContent.js

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
Extracted this `PageSpeedContent` component to prevent `DashboardPageSpeed` breaking the ESLint complexity rule. See **Relevant technical choices** in the PR description.